django_feature_flip/admin.py

```diff
@@ -2,10 +2,15 @@
 from django_feature_flip.models import Feature
 
 
+def groups_list(feature):
+    return [group.name for group in feature.groups.all()]
+
+
 class FeatureAdmin(admin.ModelAdmin):
     list_display = (
         'name',
         'totally_enabled',
+        groups_list,
```

Comment:
this groups list is the function itself, not the result.
Is that what you actually want?

Reply:
Yes, I mean, it does the trick. Would you like to have it written differently?